R/summary-log.R

```diff
@@ -266,3 +281,153 @@ do_if_bbi_sum <- function(fn, mode) {
     return(res)
   }
 }
+
+#' Calculate Akaike Information Criterion (AIC) and Bayesian Information
+#' Criterion (BIC), and append to summary log
+#' @param .log_df The output of [summary_log()]) or `run_log() %>% add_summary()`
+#' @keywords internal
+add_aic_bic <- function(.log_df){
+  .log_df %>% dplyr::mutate(
+    !!rlang::sym(AIC_COL) := 2*.data[[PARAM_COUNT_COL]] + .data[[OFV_COL]],
```

Comment:
Why is the `rlang::sym` call needed here? Doesn't `!!AIC_COL := ...` give the same result?

(Same comment applies to lines 292, 393-395, and 425-427.)